django/chat/responses.py

```diff
@@ -169,26 +170,43 @@ def summarize_response(chat, response_message):
     model = chat.options.summarize_model
 
     llm = OttoLLM(model)
+    error_str = ""
 
     if len(files) > 0:
         titles = [file.filename for file in files]
         responses = []
         for file in files:
+            if cache.get(f"stop_response_{response_message.id}", False):
+                break
             if not file.text:
-                file.extract_text(pdf_method="default")
-            if not cache.get(f"stop_response_{response_message.id}", False):
-                responses.append(
-                    summarize_long_text(
-                        file.text,
-                        llm,
-                        summary_length,
-                        target_language,
-                        custom_summarize_prompt,
-                        gender_neutral,
-                        instructions,
+                try:
+                    file.extract_text(pdf_method="default")
+                except Exception as e:
+                    error_id = str(uuid.uuid4())[:7]
+                    error_str = _(
+                        "Error extracting text from file. Try copying and pasting the text."
                     )
+                    error_str += f" _({_('Error ID')}: {error_id})_"
+                    responses.append(stream_to_replacer([error_str]))
```

Comment:
Instead of having a separate list of errors, output them under the existing filename headings